src/test/scala/com/high-performance-spark-examples/GoldiLocks/QuantileOnlyArtisanalTest.scala

```diff
@@ -1,20 +1,9 @@
 package com.highperformancespark.examples.goldilocks
 
-import org.apache.spark._
-import org.apache.spark.sql.SQLContext
-import org.scalatest.{BeforeAndAfterAll, FunSuite}
+import com.highperformancespark.examples.CustomDataFrameSuiteBase
 
 // tag::MAGIC_PANDA[]
-class QuantileOnlyArtisanallTest extends FunSuite with BeforeAndAfterAll {
```

Comment:
Whoops my bad - this was original misspelled could you fix it to QuantileOnlyArtisanalTest